### PR TITLE
Add last training of the year!?

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: resources.yaml [hunk 1]
```diff
@@ -201,3 +195,9 @@ deployment:
         start: 2022-01-12
         end: 2022-01-12
         group: training-imbs2022
+    training-nhm-:
```
Comment (bgruening) [inline resources.yaml:198]: is the leading `-` intendet?

Comment (beatrizserrano) [reply resources.yaml:198]: If you mean the last one, those (nhm-) are the 4 first characters of the id provided by the trainer (nhm-postgrad). We've had cases like this in the past and it has worked.

Comment (bgruening) [reply resources.yaml:198]: ok, cool!

**Comment by gmauro** [comment]
Illusa!

**Pull request merged**